packages/constellation/src/components/styles.css

```diff
@@ -0,0 +1,795 @@
+/*
```

Comment:
Is there a reason for doing this instead of what they suggest on their site:
```
@tailwind base;
@tailwind components;
@tailwind utilities;
```

Reply:
@noahub I see that you do use the @tailwind overrides in a file down below this one. I thought that the contents of this file were what is in the tailwind directives, so why do we need both?

Reply:
This is actually the output css generated by Tailwind. We do use those directives in the `packages/constellation/src/input.css` CSS file (along with a few customizations).

I'm open to renaming/moving these files to reduce the chances of confusion.

Reply:
Since that file is programmatically generated, I don't think we would want to check it into git right? We would defer that to a `yarn build` command or something?

Reply:
Great call @blockdylanb!